packages/react-native-nitro-modules/package.json

```diff
@@ -117,7 +117,6 @@
       [
         "typescript",
         {
-          "tsc": "../../node_modules/.bin/tsc",
```

Comment:
Adding it manually here failed on Windows.
Now that we removed it, it still works on Unix because it just recursively goes up to find `tsc`. Same story on Windows now, previously Windows didn't do that because of whatever reason.